sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -208,7 +208,7 @@ catch (RequestFailedException exception)
 ```
 For samples on using the production recommended option `AnalyzeSentimentBatch` see [here][analyze_sentiment_sample].
 
-To get more granular information about the opinions related to aspects of a product/service, also known as Aspect-based Sentiment Analysis in Natural Language Processing (NLP), see a sample on sentiment analysis with opinion mining [here][analyze_sentiment_opinion_mining_sample].
+To get more granular information about the assessment related to targets of a product/service, also known as Target-based Sentiment Analysis in Natural Language Processing (NLP), see a sample on sentiment analysis with assessment mining [here][analyze_sentiment_assessment_mining_sample].
```

Comment:
This is interesting, so the name of the feature is still `Opinion mining`. Then renames are more around the information returned when doing the opinion mining.
Because of that I suggest
```suggestion
To get more granular information about the opinions related to targets of a product/service, also known as Aspect-based Sentiment Analysis in Natural Language Processing (NLP), see a sample on sentiment analysis with opinion mining [here][analyze_sentiment_assessment_mining_sample].
```

I recommend following the same for the rest of documentation changes